<launch>
    <arg name="interest_oa_radius" default="1500" />
    <arg name="strategy" default="1" />
    <arg name="max_speed" default="1.0" />
    <arg name="max_oa_angle_speed" default="0.333" />
    <arg name="oaWeight" default="0.8" />

    <node name="avoider_cmd" pkg="tcp2ros" type="avoider_cmd" output="screen">
        <param name="interest_oa_radius" value="$(arg interest_oa_radius)" />
        <param name="strategy" value="$(arg strategy)" />
        <param name="max_speed" value="$(arg max_speed)" />
        <param name="max_oa_angle_speed" value="$(arg max_oa_angle_speed)" />
        <param name="oaWeight" value="$(arg oaWeight)" />
    </node>
</launch>
